### PR TITLE
remove rotated cards from SBL 22.08 and SBL 22.09

### PR DESCRIPTION
**Pull request by francescopellegrini**
Might as well slim them a bit

### CODE REVIEW

**Review by distributive — CHANGES_REQUESTED** [review]
Thanks for this! I totally forgot
It looks like Employee Strike accidentally got removed. Also, would you be able to update the v2 data as well? (see `v2/restrictions/standard/standard_ban_list_22_09.json` and the `22-08` equivalent)

File: mwl.json [hunk 1]
```diff
@@ -2989,33 +2938,6 @@
       "09023": {
         "deck_limit": 0
       },
-      "09053": {
-        "deck_limit": 0
-      },
```
Comment (distributive) [inline mwl.json:2994]: This is Employee Strike (Data and Destiny) which hasn't rotated

File: mwl.json [hunk 2]
```diff
@@ -3241,33 +3112,6 @@
...
```
Comment (distributive) [inline mwl.json:3246]: Same here

**Review by distributive — CHANGES_REQUESTED** [review]
Forgot to add these to the review

**Comment by francescopellegrini** [comment]
> Thanks for this! I totally forgot It looks like Employee Strike accidentally got removed.

@distributive thank you so much for pointing it out!

**Review by distributive — APPROVED** [review]
This looks good to me!

**Review by plural — APPROVED** [review]

**Pull request merged**